message_ix_models/data/water/set.yaml

```diff
@@ -370,7 +370,7 @@ nexus:
     - [loil_ppl, cooling__loil_ppl]
     - [nuc_hc, cooling__nuc_hc]
     - [nuc_lc, cooling__nuc_lc]
-    - [solar_th_ppl, cooling__solar_th_ppl]
+    # - [solar_th_ppl, cooling__solar_th_ppl]
```

Comment:
Why did you choose to comment this out? If this is not used anymore, please delete the line. If you need it again in the future, please include a comment as to why it was commented out now and when it should be enabled again. Same below in the same file (line 911).

Reply:
this is temporary, I did write a comment in the commit saying that I need to replace that solar_th_csp with the new technology names

Reply:
Great! Please also include the comment here, it may be easier to find here than in the commit message that changed the line.